Packs/CommonPlaybooks/Playbooks/playbook-Isolate_Endpoint_-_Generic_V2.yml

```diff
@@ -331,24 +304,151 @@ tasks:
       Hostname:
         simple: ${inputs.Endpoint_hostname}
     separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 2200,
+          "y": 195
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
     skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "13":
+    id: "13"
+    taskid: 40a0e4a6-9cb9-4ab4-8b66-0311dca92456
+    type: regular
     task:
-      brand: ''
-      id: d438379a-5602-49f9-8d6d-404f9dbe3919
-      iscommand: false
-      name: Microsoft Defender For Endpoint - Isolate Endpoint
-      playbookId: Microsoft Defender For Endpoint - Isolate Endpoint
-      type: playbook
+      id: 40a0e4a6-9cb9-4ab4-8b66-0311dca92456
       version: -1
-      description: ''
-    taskid: d438379a-5602-49f9-8d6d-404f9dbe3919
+      name: Cortex XSIAM - Isolate Endpoint
```

Comment:
lets change to Core - Isolate Endpoint since XDR is not mandatory within XSIAM

Reply:
Done